src/app/settings/views/Security/IpmiSettings/IpmiFormFields/IpmiFormFields.tsx

```diff
@@ -0,0 +1,76 @@
+import { Link } from "@canonical/react-components";
+
+import FormikField from "app/base/components/FormikField";
+import TooltipButton from "app/base/components/TooltipButton";
+import docsUrls from "app/base/docsUrls";
+
+export enum Labels {
+  IPMIUsername = "MAAS generated IPMI username",
+  KGBMCKeyLabel = "Auto IPMI K_g BMC key",
+  UserRadio = "User",
+  AdminRadio = "Admin",
+  OperatorRadio = "operator",
```

Comment:
Prior to the RTL refactor[^1] the operator label used to start with capital letter to match User and Admin[^2].

```suggestion
  OperatorRadio = "Operator",
```
[^1]: https://github.com/canonical/maas-ui/blame/43ef634be6a62d7f998b5ab829ef2669d0c2495b/src/app/settings/views/Configuration/CommissioningFormFields/CommissioningFormFields.tsx#L21
[^2]: https://github.com/canonical/maas-ui/blame/b6bc560c68856b6a29c355cd94e6d82c74572483/src/app/settings/views/Configuration/CommissioningFormFields/CommissioningFormFields.tsx#L108